.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: generic
+language: c
```

Comment:
Sounds good.  If gcc is the expectation may not hurt to specify it but no biggie either way!